SeaPublicWebsite/appsettings.json

```diff
@@ -8,8 +8,13 @@
     },
     "AllowedHosts": "*",
     
-    "OpenEpcBaseAddress": "https://epc.opendatacommunities.org",
-    "EpbEpcBaseAddress": "https://api.epb-staging.digital.communities.gov.uk",
+    "OpenEpc": {
+        "BaseUrl": "https://epc.opendatacommunities.org"
+    },
+    
+    "EpbEpc": {
+        "BaseUrl": "https://api.epb-staging.digital.communities.gov.uk"
+    },
     

```

Comment:
And the Bre base url needs to be added here (it's https://uat.brewebserv.com but happy to add it myself later)

Reply:
I've added this :)